### PR TITLE
[WIP] relay closeChanConfirm

### PR DESCRIPTION
**Pull request by ancazamfir**
To address  #5

### CODE REVIEW

**Review by jackzampolin — APPROVED** [review]
Looking good so far @ancazamfir! Thank you for cleaning up my mistakes to 🙊

**Comment by ancazamfir** [comment]
I added `relayer tx channel-close ibc0 ibc1` command that checks if a channel end is in `CLOSED` state and relays the `ChannCloseConfirm`. To initiate the connection close the raw command may be used: `relayer tx raw chan-close-init ibc0 ibconexfer transfer`. 
Should I change the `relayer tx channel-close` to take a parameter and do the initiation as well?

**Comment by jackzampolin** [comment]
@ancazamfir yes please! the command `relayer tx channel-close [src] [dst]` should both initiate and finish the closing. If someone needs the steps separated, they should use the raw commands.

File: cmd/tx.go [hunk 1]
```diff
@@ -180,6 +215,9 @@ func fullPathCmd() *cobra.Command {
 			// NOTE: this is hardcoded to create ordered channels right now. Add a flag here to toggle
 			// Check if channel has been created, if not create it
 			return chains[src].CreateChannel(chains[dst], true, to)
+
+			// Check if channel has been created, if not create it
+			return chains[src].CreateChannel(chains[dst], true, to)
```
Comment (jackzampolin) [inline cmd/tx.go]: This looks extraneous.

**Review by jackzampolin — APPROVED** [review]
One nit, but G2G otherwise.

File: cmd/tx.go (hunk 1 above)
Comment (ancazamfir) [reply cmd/tx.go]: ouch, missed that, fixed now.

**Comment by zmanian** [comment]
@ancazamfir do you want me push a fix for the merge conflicts?

**Comment by ancazamfir** [comment]
> @ancazamfir do you want me push a fix for the merge conflicts?


Sure, if you have time now thanks! Otherwise I will do it later today.

**Pull request merged**